contrib/openshift/osba-os-template.yaml

```diff
@@ -0,0 +1,238 @@
+    template:
+      metadata:
+        labels:
+          app: osba-open-service-broker-azure
+          release: "osba"
+      spec:
+        containers:
+          - name: open-service-broker-azure
+            image: microsoft/azure-service-broker:v0.10.0
+            imagePullPolicy: IfNotPresent
+            env:
+            - name: ENVIRONMENT
+              value: ${ENVIRONMENT}
+            - name: AZURE_SUBSCRIPTION_ID
+              value: ${AZURE_SUBSCRIPTION_ID}
+            - name: AZURE_TENANT_ID
+              value: ${AZURE_TENANT_ID}
+            - name: AZURE_CLIENT_ID
+              value: ${AZURE_CLIENT_ID}
+            - name: AZURE_CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: osba-open-service-broker-azure
+                  key: azure-client-secret
+            - name: AZURE_DEFAULT_LOCATION
+              value: ${AZURE_DEFAULT_LOCATION}
+            - name: REDIS_HOST
+              value: osba-redis
+            - name: REDIS_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: osba-open-service-broker-azure
+                  key: redis-password
+            - name: AES256_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: osba-open-service-broker-azure
+                  key: encryption-key
+            - name: BASIC_AUTH_USERNAME
+              value: username
+            - name: BASIC_AUTH_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: osba-open-service-broker-azure
+                  key: basic-auth-password
+            - name: MIN_STABILITY
+              value: EXPERIMENTAL
+            ports:
+            - containerPort: 8080
+            readinessProbe:
+              tcpSocket:
+                port: 8080
+              failureThreshold: 1
+              initialDelaySeconds: 10
+              periodSeconds: 10
+              successThreshold: 1
+              timeoutSeconds: 2
+            livenessProbe:
+              tcpSocket:
+                port: 8080
+              failureThreshold: 3
+              initialDelaySeconds: 30
+              periodSeconds: 10
+              successThreshold: 1
+              timeoutSeconds: 2
+- apiVersion: servicecatalog.k8s.io/v1beta1
+  kind: ClusterServiceBroker
+  metadata:
+    name: osba
+  spec:
+    url: http://osba-open-service-broker-azure.osba.svc.cluster.local
+    authInfo:
+      basic:
+        secretRef:
+          name: "osba-open-service-broker-azure-auth"
+          namespace: "osba"
+
+parameters:
+- name: ENVIRONMENT
+  value: AzurePublicCloud
+  required: true
+- name: AZURE_SUBSCRIPTION_ID
+  value: 
+  required: true
+- name: AZURE_TENANT_ID
+  value: 
+  required: true
+- name: AZURE_CLIENT_ID
+  value: 
+  required: true
+- name: AZURE_DEFAULT_LOCATION
+  value: 
+  required: true
+- name: AZURE_CLIENT_SECRET
+  value: 
+  required: true
```

Comment:
It seems to me that there are a lot of missing parameters here. Without more parameters, users cannot do things like run OSBA with a remote Redis or, much more importantly, override the default password or encryption keys. This needs to be addressed as soon as possible.